Woodoku/src/entity/woodoku_shape.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 from typing import List, Tuple, Set
+from ui.utils import *
```

Comment:
```suggestion
from ui.utils import green, BLOCK
```
only import what is necessary